universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx

```diff
@@ -56,6 +56,10 @@ export const TopUpWithFiat = ({sdk, onPayClick, logoColor}: TopUpWithFiatProps)
 
   useAsyncEffect(recognizeUserCountry, []);
 
+  const isPayButtonDisabled = () =>
```

Comment:
What's the reasoning behind using a function here instead of just a plain constant?